src/types/directory.ts

```diff
@@ -38,6 +42,14 @@ export interface MediaData {
   type: "file"
 }
 
+// NOTE: This assumes that mediaPath is proof that this is a media file.
+// The corresponding backend code (mediaFiles.js) does not appear to guarantee the existence of
```

Comment:
Hmm do you happen to have an example where the backed code returns a mediaPath without having a media file? While it would be beyond the scope of this PR - I feel that we can minimally document this bug as an issue as this front-end fix seems to me like a temporary workaround